Pomodoro/Sources/View/ViewController.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  ViewController.swift
+//  Pomodoro
+//
+//  Created by Радик Ахметзянов on 20.08.2022.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    // MARK: - Properties
+        
+    var isStarted = false
+    var isWorkTime = false
+    var timer: Timer!
```

Comment:
Принудительное извлечение
```suggestion
    var timer = Timer()
```